roottest/root/treeformula/event/CMakeLists.txt

```
@@ -1,7 +1,49 @@
-#-------------------------------------------------------------------------------
-#
-# Placeholder file to translate the tests to the new CTest system. Meanwhile we
-# define a CTest test that calls 'make' in ${CMAKE_CURRENT_SOURCE_DIR}
-#
-#-------------------------------------------------------------------------------
-ROOTTEST_ADD_OLDTEST()
+set(_size 50)
+set(_tracks 600)
+set(_comp 0)
+set(_action 1)
+
+ROOTTEST_COMPILE_MACRO(dt_RunDrawTest.C
+                       FIXTURES_SETUP root-treeformula-event-RunDrawTest-fixture)
+
+foreach(split 0 1 9 -2 -1)
+  set(_smode new)
+  set(_sname ${split})
+  if(${split} EQUAL -1)
+    set(_smode old)
+    set(_sname streamed)
+  elseif(${split} EQUAL -2)
+    set(_smode old)
+    set(_sname split)
+  endif()
+
+  set(_file_name  Event.${_smode}.split${_sname}.root)
+  ROOTTEST_ADD_TEST(write-${_smode}-${_sname}
+                    COMMAND ${CMAKE_BINARY_DIR}/roottest/root/io/event/TestIoEvent ${_size} ${_comp} ${split} ${_action} ${_tracks} ${_file_name}
+                    FIXTURES_REQUIRED root-io-event-TestIoEvent-fixture
+                    FIXTURES_SETUP root-treeformula-event-write-${_sname}-fixture)
```

```suggestion
                    FIXTURES_SETUP root-treeformula-event-write-${_smode}-${_sname}-fixture)
```
might be better (allow more parallelism).